karma.conf.js

```diff
@@ -15,10 +15,10 @@ module.exports = (config) => {
 
   // Override process.env.NODE_ENV
   webpackConfig.plugins[1] = new webpack.DefinePlugin({
-      'process.env': {
-        NODE_ENV: JSON.stringify('test')
-      }
-    });
+    'process.env': {
+      NODE_ENV: JSON.stringify('test')
```

Comment:
The general convention here seems to be to use `'production'` instead of other strings.